go/vt/wrangler/materializer.go

```diff
@@ -741,16 +759,22 @@ func (wr *Wrangler) prepareCreateLookup(ctx context.Context, keyspace string, sp
 	materializeQuery = buf.String()
 
 	// Update targetVSchema
-	var targetTable *vschemapb.Table
+	targetTable := specs.Tables[targetTableName].CloneVT()
```

Comment:
Why do a clone here?

Reply:
Wanted to make sure it doesn’t end up modifying the original here from the user input.

Reply:
We don't change it though. And I feel like if we do change it, that might be what we want? I don't know why we'd want them to differ off the top of my head. I was gong to skip this part in the vtctldclient implementation.